Lib/test/test_sys_settrace.py

```diff
@@ -2064,6 +2064,40 @@ def test_jump_simple_backwards(output):
         output.append(1)
         output.append(2)
 
+    @jump_test(1, 4, [2])
+    def test_jump_is_none_forwards(output):
+        x = None
+        if x is None:
+            output.append(1)
+        else:
+            output.append(2)
+
+    @jump_test(2, 1, [1, 3])
+    def test_jump_is_none_backwards(output):
+        x = None
+        if x is None:
+            output.append(1)
+        else:
+            output.append(2)
+        output.append(3)
```

Comment:
I'm pretty sure this test (and `test_jump_is_not_none_backwards`, below) don't actually fail on `main` right now. I think to trigger this bug, you need to jump into the `else` clause.

So maybe jump from 6 to 5, and expect an output of `[1, 2, 3]` (or `[3, 5, 6]`, if we update this test to use line numbers).